geoblacklight.gemspec

```diff
@@ -30,6 +30,7 @@ Gem::Specification.new do |spec|
   spec.add_dependency 'mime-types'
   spec.add_dependency 'handlebars_assets'
   spec.add_dependency 'rgeo-geojson'
+  spec.add_dependency 'view_component', '>= 2.28.0'
```

Comment:
Not sure we need this as view_component is a dependency of blacklight since [v7.8.0](https://github.com/projectblacklight/blacklight/blob/v7.8.0/blacklight.gemspec). They require 2.28.0 as of [v7.17.0](https://github.com/projectblacklight/blacklight/blob/v7.17.0/blacklight.gemspec)

Reply:
Sure! Actually, I'll bump the BL dependency up to 7.8.0 here to pick up ViewComponents.